scripts/utils.js

```diff
@@ -32,9 +32,34 @@ function getIdForStatusWithName(statusName, possibleActionsList) {
     return null;
 }
 
+//find status id by statusName among possibleActionsList
+function getIdForStatusWithNameIgnoreCase(statusName, possibleActionsList) {
+    var actionId = 0;
+    for each (var actionDescriptor in possibleActionsList) {
+       	if(actionDescriptor.getName().toLowerCase() === statusName) {
```

Comment:
Indents...

Please set up IDE to use only spaces...

Reply:
I don't see this tab in my code and I have the same version of code as in repo.
I see this tab on this page https://github.com/adokaeva/jira-git-workflow-hooks/blob/GIT-3009/scripts/utils.js but I don't see this tab when  I try to edit this page from browser.